docs/planning/bug_triage.md

```diff
@@ -0,0 +1,44 @@
+# Knative Eventing Bug Triage and Management
+
+This document summarizes how bugs will be triaged and managed in Knative Eventing WG.
+This is a supplement to [Sprint planning](sprintplanning.md) process.
+
+## Goal
+
+- Keep a low bug count. Bug count should not be consistently increasing release over release esp [priority/important-longterm](#Bug-priorities).
+- Process shouldn't be taxing for a single person or the team.
+- Maintain high product quality.
+
+## Bug priorities
+
+- priority/critical-urgent: Bugs that need to be fixed now, and possibly cherrypicked into the most recent release. May need patch-fix.
+- priority/important-soon: Bugs that need to be fixed in the next release cycle, and but do not need a patch-fix.
+- priority/important-longterm: Bugs that need to be fixed, and will be fixed in next 2-3 releases.
+- priority/awaiting-more-evidence: Bugs which might need to be fixed, but need more details first.
+
+## How to manage bugs?
+
+- Submit new bugs anytime by creating a new issue of type “Bug”.
+- Please fill the bug template and optionally propose a priority.
```

Comment:
I would not add this are the end user opening the bug might not know the priority, but the template will have the right set of questions to gather such
- This used to work in a previous release and it doesn't (ie regression)
- Is this issue a blocker and you need immediate help with a workaround until the fix is available in the next release (ie urgent)